src/api/routes/company.js

```diff
@@ -57,4 +61,40 @@ module.exports = (app) => {
         }
 
     });
+
+    router.post(
+        "/:companyId/block",
+        or([
+            authMiddleware.isGod,
+            authMiddleware.isAdmin
+        ],
+        { status_code: HTTPStatus.UNAUTHORIZED, error_code: ErrorTypes.FORBIDDEN, msg: ValidationReasons.INSUFFICIENT_PERMISSIONS }),
+        validators.manage,
```

Comment:
why manage and not block?

Reply:
So it could also be used in disable route, probably not the best name.

Reply:
I mean unblock